content/docs/ops/secrets-dependencies.md

```diff
@@ -0,0 +1,59 @@
+---
+menu:
+  docs:
+    parent: operations
+layout: ops
+title: Secrets Dependencies
+---
+
+This page exists to contain the secrets documentation for the dependencies
+across all of our deployments. As more of our secrets are automated, they will
+be address / removed from this age as resolved.
+
+### Certificate dependencies
+
+{{< diagrams id-prefix="certificates-depending-on-single-root-certificate" >}}
+graph TB;
+    %% ID Definition
+
+    s3(s3://cloud-gov-varz)
```

Comment:
How about a little less explicit, like 
```
s3(s3://secrets-bucket)
```

Reply:
If these docs were long-lived, I'd say that it may make sense to be generic, but we will have to be able to differentiate between the various buckets we have secrets in. Also this information is public in our `cg-provision` repo and are very likely never going to change unless we want to change _all the things_. 

https://github.com/18F/cg-provision/search?utf8=✓&q=cloud-gov-varz